contracts/staking/validatorShare/ValidatorShare.sol

```diff
@@ -121,6 +120,8 @@ contract ValidatorShare is IValidatorShare {
             _amount = _amount.sub(_rewards);
         }
 
+        activeAmount = activeAmount.sub(_amount);
```

Comment:
Please explain what changed in `activeAmount` before and after this change?

It would be great if we can have scenarios in PR comments where it was breaking and where it was working, to track the timeline of the problem.